package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "simple-report",
+  "version": "0.1.0",
+  "private": true,
+  "devDependencies": {
+    "husky": "4",
+    "lerna": "^3.22.1"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lerna run --concurrency 1 --stream precommit"
```

Comment:
`lerna` kinda seems like overkill to me since we only have one module using the node toolchain. What if we went with something like `$(cd frontend && npm run precommit) && $(cd backend && ./gradlew spotlessApply)`? That would also let us leverage `terraform fmt` for assets in the `ops` directory tree.

Reply:
Something like this could work as well - the only reason I added `lint-staged` for the backend as well was so that spotless would only get run against staged files instead of all the files. But the time difference is probably minimal, so I'm happy to take this simpler route

Reply:
The other nice thing `lerna` provides is the nice log output you see in the screenshot above - but I think you're right that it's probably not worth the extra cruft